d20potz.py

```diff
@@ -378,23 +379,22 @@ async def cards_command(update: Update, context: ContextTypes.DEFAULT_TYPE):
         )
         return
     elif sub_command == "hand":
-        try:
-            active_cards = list(
-                get_player_cards(update.effective_chat.id, player_name, flipped=False)
-            )
-            await send_cards(player_name, active_cards, chat_id, context)
-            flipped_cards = list(
-                get_player_cards(update.effective_chat.id, player_name, flipped=True)
-            )
-            await context.bot.send_message(
-                chat_id=chat_id,
-                text="{}'s flipped cards {}".format(player_name, flipped_cards),
-            )
-        except:
-            await context.bot.send_message(
-                chat_id=chat_id,
-                text="{} has no cards.".format(spell_hero_name(player_name)),
-            )
+        active_cards = list(
```

Comment:
just removing try/except here won't solve the issue, get_player_cards throws if it does not find any entry with that key.
I propose moving try/catch there and returning an empty list in that case

Reply:
There is really no need for try/catch, code should work with empty inputs.